app/uk/gov/hmrc/hecstubs/controllers/IndividualAccountOverviewController.scala

```diff
@@ -41,10 +41,14 @@ class IndividualAccountOverviewController @Inject() (cc: ControllerComponents)
   val yearRegex         = "^[0-9]{4}$"
 
   /**
-    * fetch the individual account overview based on utr and tx year
-    * @param utr
-    * @param taxYear year the tax year ends
-    * @return
+    * Fetch the individual account overview based on utr and tax year.
+    * The default status returned is ReturnFound.
+    * Provide UTR starting with
+    *  - "1111" to receive a status of NoReturnFound
+    *  - "2222" to receive a status of NoticeToFileIssued
+    * @param utr The individual's UTR
```

Comment:
a note here to say that everything else returns a `ReturnFound` response would also be handy

Reply:
There's a comment above that says `The default status returned is ReturnFound.`

Reply:
whoops, sorry - missed that :laughing: